src/vivarium_gates_mncnh/data/extra_gbd.py

```diff
@@ -16,33 +18,51 @@
 def get_maternal_disorder_yld_rate(key: str, location: str) -> pd.DataFrame:
     entity = utilities.get_entity(key)
     location_id = utility_data.get_location_id(location)
-    data = vi_utils.get_draws(
+    data = get_draws(
         "cause_id",
         entity.gbd_id,
         source=gbd_constants.SOURCES.COMO,
         location_id=location_id,
         year_id=ARTIFACT_YEAR_START,
-        release_id=gbd_constants.RELEASE_IDS.GBD_2021,
+        release_id=gbd_constants.RELEASE_IDS.GBD_2023,
         measure_id=vi_globals.MEASURES["YLDs"],
         metric_id=vi_globals.METRICS["Rate"],
     )
     return data
 
 
 @vi_utils.cache
-def load_lbwsg_exposure(location: str) -> pd.DataFrame:
+def load_2021_lbwsg_birth_exposure(location: str) -> pd.DataFrame:
     entity = utilities.get_entity(data_keys.LBWSG.BIRTH_EXPOSURE)
     location_id = utility_data.get_location_id(location)
-    data = vi_utils.get_draws(
+    data = get_draws(
         gbd_id_type="rei_id",
         gbd_id=entity.gbd_id,
         source=gbd_constants.SOURCES.EXPOSURE,
         location_id=location_id,
-        year_id=ARTIFACT_YEAR_START,
+        year_id=ARTIFACT_YEAR_START - 1,
```

Comment:
I'd rather you just hardcode 2022 rather than do this. We don't actually want 1 less than the artifact start year - we just don't have any data newer than 2022